pkgs/by-name/pw/pwsafe-cli/package.nix

```
@@ -0,0 +1,65 @@
+{ lib,
+stdenv,
```

```suggestion
{ lib
, stdenv
```
I don't like to format by hand, but...